Automattic/wp-calypso
# Jetpack Cloud: Don't show backup error when a steam activity has occurred after last backup

#### Changes proposed in this Pull Request

* `lastBackup` property can now only be a rewindable activity, to prevent it being picked up as a failed backup

#### Testing instructions

1. Add a stream activity like a multiple media upload to a test site with **daily backups**
2. Navigate to `/backups` for the same test site on `cloud.jetpack.com` and verify it is now displaying a failed backup
3. Navigate to `/backups` on this branch and verify it is now showing the correct status for the backups
4. Navigate to `/backups` for a site with **real-time backups**, confirm that nothing has changed vs production.
